code/screenshot.py

```diff
@@ -42,6 +42,14 @@ def screenshot_selection():
         elif app.platform == "linux":
             actions.key("shift-printscr")
 
+    def screenshot_settings():
+        """Opens the settings UI for screenshots.
+        Only applies to Mac for now
+        """
+        if app.platform == "mac":
+            actions.key("cmd-shift-5")
```

Comment:
Please add an `else` branch that raises an error if we're not on mac.

Reply:
@rntz I've now added this. I can't test it due to my OS, but the code is pretty basic.